### PR TITLE
Create ADAGhostchain

### PR DESCRIPTION
**Pull request by ejane24**
proof will be posted here -> https://twitter.com/AdaGhostSociety

### CODE REVIEW

File: projects/ADAGhostchain [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "project": "ADA Ghostchain",
```
Comment (kriscarilloxyz) [inline projects/ADAGhostchain:3]: Hey man, it's **ADA GhostChain Society**

**Review by ejane24 — COMMENTED** [review]
update name

**Comment by KaiNFT** [comment]
Hi @ejane24 

Apologies for the long delay, however the policy ID you've posted doesn't match with the two ID's listed in your discord.

**Comment by kriscarilloxyz** [comment]
Kindly close, already merged @KaiNFT  #5108

**Pull request closed without merging**